snippets/paths.py

```diff
@@ -1,47 +1,101 @@
-"""Example of how to find the best path to trade with"""
+"""Example of ripple_path_find request using trustlines and rippling"""
+
 from xrpl.clients import JsonRpcClient
-from xrpl.models import XRP, IssuedCurrencyAmount, Payment, RipplePathFind
-from xrpl.transaction import autofill_and_sign
+from xrpl.models import AccountSet, TrustSet
+from xrpl.models.amounts import IssuedCurrencyAmount
+from xrpl.models.requests import RipplePathFind
+from xrpl.models.transactions.account_set import AccountSetAsfFlag
+from xrpl.transaction import submit_and_wait
 from xrpl.wallet import generate_faucet_wallet
 
-# References
-# - https://xrpl.org/paths.html#paths
-# - https://xrpl.org/ripple_path_find.html#ripple_path_find
+# Note: This test is inspired from a unit test titled `indirect_paths_path_find` in the
```

Comment:
The main purpose of a snippet is providing a working example for users, not for testing even though we implement tests on those. This should be reworded